docs/en/latest/plugins/mqtt-proxy.md

```diff
@@ -23,10 +23,11 @@ title: mqtt-proxy
 
 ## Summary
 
-- [**Name**](#name)
-- [**Attributes**](#attributes)
-- [**How To Enable**](#how-to-enable)
-- [**Delete Plugin**](#delete-plugin)
+- [Summary](#summary)
```

Comment:
We don't need to add an anchor for summary?

Reply:
You're right.

Reply:
We don't need to change the summary in this PR?